examples/quickstart/new_quickstart/new_quickstart.ipynb

```diff
@@ -0,0 +1,1471 @@
+{
```

Comment:
<p>This quickstart demonstrates some of ZenML's features. We will: </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zenml-io/zenml/pull/1515/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>